ynput/OpenPype
# Fusion: automatic installation of PySide2

## Changelog Description
This PR adds hook which tries to check if PySide2 is installed in Python used by Fusion and if not, it tries to install it automatically.

## Additional info
This is customer requirement. Even that there is explicit (which stays there) menu option inside of Fusion, it is happening that artists will not press it.
Hook is based on Blender approach, but checking for PySide2 is simplified as there is some issue about missing `dataclass` for click in Python3.6

## Testing notes:
1. uninstall PySide2 from Python 3.6 used by Fusion (you could learn where that Python is located directly in Fusion console:
![image](https://github.com/ynput/OpenPype/assets/4457962/03a9ddca-f927-4adc-a8e1-bd5f7e1a644b)
2. launch Fusion - it should install PySide2
3. launch Fusion again - it shouldn't do anything
